src/controllers/aer_controller.hpp

```diff
@@ -86,17 +86,19 @@ class Controller {
   template <typename inputdata_t>
   Result execute(const inputdata_t &qobj);
 
+  template <class config_t>
   Result execute(std::vector<Circuit> &circuits,
                  Noise::NoiseModel &noise_model,
-                 const json_t &config);
+                 const config_t &config);
```

Comment:
I added this template type `config_t` to allow configuration from `py:handle`. However, current pybind11 does not provide thread-safe access to python's data structure from C++ even though the access is read. Actually, I observed several segmentation faults in `set_config(config)` in OMP loop. I believe that this issue will be resolved in near future, so I keep this template type.